src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+.img-responsive {
+  width: 100%;
+}
+.list-none {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+.word-break{
+  word-break: break-word;
+}
+.d-block {
+  display: block;
+}
+.no-padding{
+  padding: 0 !important;
+}
+input[type="submit"],
+button {
+  cursor: pointer;
+}
+img {
+  vertical-align: top;
+  border-style: none;
+}
+button:focus,
+textarea:focus,
+input:focus,
+select {
+  outline: none;
+}
+textarea,
+input[type="text"],
+input[type="email"],
+input[type="password"],
+input[type="tel"],
+input[type="number"],
+input[type="date"],
+select{
+  color:#000;
+}
+input[type="date"]::-webkit-calendar-picker-indicator{
+  background: transparent;
+  bottom: 0;
+  color: transparent;
+  cursor: pointer;
+  height: auto;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: auto;
+}
+body {
+  margin: 0;
+  color: var(--body-text-color);
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: none;
+  font: 400 16px/20px "Poppins", sans-serif;

```

Comment:
I think we should use web safe fonts instead of a google font because font family is design dependent.